akka-sample-distributed-data-scala/src/main/scala/sample/distributeddata/ServiceRegistry.scala

```diff
@@ -13,10 +13,9 @@ import akka.cluster.ddata.GSet
 import akka.cluster.ddata.GSetKey
 import akka.cluster.ddata.Key
 import akka.cluster.ddata.ORSet
+import com.fasterxml.jackson.annotation.JsonCreator;
 
 object ServiceRegistry {
```

Comment:
let's remove this example, because it's exactly what the `Receptionist` is about